www_src/positionable.less

```diff
@@ -0,0 +1,88 @@
+.project {
+  position: relative;
+
+  .fakelink {
+    color: blue;
+    text-decoration: underline;
+  }
+
+  .controls {
+    position: absolute;
+    bottom: 2em;
+    left: 0;
+    right: 0;
+    z-index: 999;
+    margin: auto;
+    width: 6em;
+    button {
+      color: rgb(10,110,255);
+      border-radius: 1em;
+      border: 1px solid black;
+      padding: 0 2px;
+      background: white;
+      box-shadow: 0 0 3px 4px rgba(0,0,0,0.1);
+      width: 2em;
+      height: 2em;
+      overflow: hidden;
+      &.image.component {
+        position: relative;
+        top: -1em;
+      }
+    }
+  }
+
+  .shadows {
+    position: absolute;
+    height: 0;
+    opacity: 0.2;
+    margin-left: -5px;
+
+    -webkit-filter: grayscale(100%);
```

Comment:
We're using autoprefixer so you can just use the standard property names. :dancer: